testing/file_test/file_test_base.cpp

```diff
@@ -720,18 +725,33 @@ static auto Main(int argc, char** argv) -> int {
     return EXIT_FAILURE;
   }
 
+  // Tests might try to read from stdin. Ensure those reads fail by closing
+  // stdin and reopening it as /dev/null. Note that STDIN_FILENO doesn't exist
+  // on Windows, but POSIX requires it to be 0.
+  llvm::sys::Process::SafelyCloseFileDescriptor(/*STDIN_FILENO*/ 0);
```

Comment:
```suggestion
  llvm::sys::Process::SafelyCloseFileDescriptor(0);
```

Comment syntax isn't correct for an argument comment.